DEBUG - 2025-06-05 00:03:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:13:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:13:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:13:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:13:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 00:23:10 --> ParseError: syntax error, unexpected token ";", expecting "]"
[Method: GET, Route: clientes]
in APPPATH\Controllers\Clientes.php on line 69.
 1 SYSTEMPATH\Autoloader\Autoloader.php(295): CodeIgniter\Autoloader\Autoloader->includeFile('C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Clientes.php')
 2 SYSTEMPATH\Autoloader\Autoloader.php(271): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers\\Clientes')
 3 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers\\Clientes')
 4 SYSTEMPATH\CodeIgniter.php(885): class_exists('\\App\\Controllers\\Clientes', true)
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->startController()
 6 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 8 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 9 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
10 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 00:24:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:27:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:29:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:29:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:29:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:29:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:53:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:53:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:53:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:53:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:53:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:53:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:53:30 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:53:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:53:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:53:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:53:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:53:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:53:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:53:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:53:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:54:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:54:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:54:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:54:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 00:54:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 01:08:24 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 01:08:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 01:08:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 01:08:35 --> ErrorException: Undefined array key 0
[Method: GET, Route: usuarios/edit/assets/css/style.css]
in APPPATH\Controllers\Usuarios.php on line 114.
 1 APPPATH\Controllers\Usuarios.php(114): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 0', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Usuarios.php', 114)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->edit('assets', 'css', 'style.css')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 01:08:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 01:08:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-06-05 01:10:08 --> mysqli_sql_exception: Unknown column 'clientes.endereco' in 'field list' in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `vendas`...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `vendas`...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `vendas`...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `vendas`...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter4\app\Controllers\Vendas.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Vendas->index()
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\codeigniter4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2025-06-05 01:10:08 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'clientes.endereco' in 'field list'
[Method: GET, Route: vendas]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `vendas`.*, `clientes`.`endereco`, `usuarios`.`usuarios_nome`
FROM `vendas`
JOIN `clientes` ON `clientes`.`usuario_id` = `vendas`.`cliente_id`
JOIN `usuarios` ON `usuarios`.`usuarios_id` = `vendas`.`cliente_id`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Vendas.php(14): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-05 01:10:08 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'clientes.endereco' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `vendas`.*, `clientes`.`endereco`, `usuarios`.`usuarios_nome`
FROM `vendas`
JOIN `clientes` ON `clientes`.`usuario_id` = `vendas`.`cliente_id`
JOIN `usuarios` ON `usuarios`.`usuarios_id` = `vendas`.`cliente_id`')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `vendas`.*, `clientes`.`endereco`, `usuarios`.`usuarios_nome`
FROM `vendas`
JOIN `clientes` ON `clientes`.`usuario_id` = `vendas`.`cliente_id`
JOIN `usuarios` ON `usuarios`.`usuarios_id` = `vendas`.`cliente_id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `vendas`.*, `clientes`.`endereco`, `usuarios`.`usuarios_nome`
FROM `vendas`
JOIN `clientes` ON `clientes`.`usuario_id` = `vendas`.`cliente_id`
JOIN `usuarios` ON `usuarios`.`usuarios_id` = `vendas`.`cliente_id`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Vendas.php(14): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-05 01:10:08 --> [Caused by] mysqli_sql_exception: Unknown column 'clientes.endereco' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `vendas`.*, `clientes`.`endereco`, `usuarios`.`usuarios_nome`
FROM `vendas`
JOIN `clientes` ON `clientes`.`usuario_id` = `vendas`.`cliente_id`
JOIN `usuarios` ON `usuarios`.`usuarios_id` = `vendas`.`cliente_id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `vendas`.*, `clientes`.`endereco`, `usuarios`.`usuarios_nome`
FROM `vendas`
JOIN `clientes` ON `clientes`.`usuario_id` = `vendas`.`cliente_id`
JOIN `usuarios` ON `usuarios`.`usuarios_id` = `vendas`.`cliente_id`')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `vendas`.*, `clientes`.`endereco`, `usuarios`.`usuarios_nome`
FROM `vendas`
JOIN `clientes` ON `clientes`.`usuario_id` = `vendas`.`cliente_id`
JOIN `usuarios` ON `usuarios`.`usuarios_id` = `vendas`.`cliente_id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `vendas`.*, `clientes`.`endereco`, `usuarios`.`usuarios_nome`
FROM `vendas`
JOIN `clientes` ON `clientes`.`usuario_id` = `vendas`.`cliente_id`
JOIN `usuarios` ON `usuarios`.`usuarios_id` = `vendas`.`cliente_id`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Vendas.php(14): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->index()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
ERROR - 2025-06-05 01:10:13 --> mysqli_sql_exception: Unknown column 'clientes.endereco' in 'field list' in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `vendas`...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `vendas`...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `vendas`...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `vendas`...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\system\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\codeigniter4\app\Controllers\Vendas.php(14): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Vendas->index()
#8 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\codeigniter4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#12 C:\xampp\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#13 C:\xampp\htdocs\codeigniter4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#14 {main}
CRITICAL - 2025-06-05 01:10:13 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'clientes.endereco' in 'field list'
[Method: GET, Route: vendas]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `vendas`.*, `clientes`.`endereco`, `usuarios`.`usuarios_nome`
FROM `vendas`
JOIN `clientes` ON `clientes`.`usuario_id` = `vendas`.`cliente_id`
JOIN `usuarios` ON `usuarios`.`usuarios_id` = `vendas`.`cliente_id`', [], false)
 2 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 3 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 4 APPPATH\Controllers\Vendas.php(14): CodeIgniter\BaseModel->findAll()
 5 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->index()
 6 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 7 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 8 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 9 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
10 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
11 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-05 01:10:13 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'clientes.endereco' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `vendas`.*, `clientes`.`endereco`, `usuarios`.`usuarios_nome`
FROM `vendas`
JOIN `clientes` ON `clientes`.`usuario_id` = `vendas`.`cliente_id`
JOIN `usuarios` ON `usuarios`.`usuarios_id` = `vendas`.`cliente_id`')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `vendas`.*, `clientes`.`endereco`, `usuarios`.`usuarios_nome`
FROM `vendas`
JOIN `clientes` ON `clientes`.`usuario_id` = `vendas`.`cliente_id`
JOIN `usuarios` ON `usuarios`.`usuarios_id` = `vendas`.`cliente_id`')
 3 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `vendas`.*, `clientes`.`endereco`, `usuarios`.`usuarios_nome`
FROM `vendas`
JOIN `clientes` ON `clientes`.`usuario_id` = `vendas`.`cliente_id`
JOIN `usuarios` ON `usuarios`.`usuarios_id` = `vendas`.`cliente_id`', [], false)
 4 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 5 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 6 APPPATH\Controllers\Vendas.php(14): CodeIgniter\BaseModel->findAll()
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->index()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-05 01:10:13 --> [Caused by] mysqli_sql_exception: Unknown column 'clientes.endereco' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('SELECT `vendas`.*, `clientes`.`endereco`, `usuarios`.`usuarios_nome`
FROM `vendas`
JOIN `clientes` ON `clientes`.`usuario_id` = `vendas`.`cliente_id`
JOIN `usuarios` ON `usuarios`.`usuarios_id` = `vendas`.`cliente_id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `vendas`.*, `clientes`.`endereco`, `usuarios`.`usuarios_nome`
FROM `vendas`
JOIN `clientes` ON `clientes`.`usuario_id` = `vendas`.`cliente_id`
JOIN `usuarios` ON `usuarios`.`usuarios_id` = `vendas`.`cliente_id`')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `vendas`.*, `clientes`.`endereco`, `usuarios`.`usuarios_nome`
FROM `vendas`
JOIN `clientes` ON `clientes`.`usuario_id` = `vendas`.`cliente_id`
JOIN `usuarios` ON `usuarios`.`usuarios_id` = `vendas`.`cliente_id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1649): CodeIgniter\Database\BaseConnection->query('SELECT `vendas`.*, `clientes`.`endereco`, `usuarios`.`usuarios_nome`
FROM `vendas`
JOIN `clientes` ON `clientes`.`usuario_id` = `vendas`.`cliente_id`
JOIN `usuarios` ON `usuarios`.`usuarios_id` = `vendas`.`cliente_id`', [], false)
 5 SYSTEMPATH\Model.php(286): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\BaseModel.php(675): CodeIgniter\Model->doFindAll(0, 0)
 7 APPPATH\Controllers\Vendas.php(14): CodeIgniter\BaseModel->findAll()
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Vendas->index()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Vendas))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
14 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 01:10:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 01:10:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 16:49:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 16:49:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 16:50:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 16:50:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 16:50:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 16:50:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 16:50:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 16:50:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 16:50:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 16:51:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:01:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:01:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:10:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:13:32 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:14:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-06-05 17:16:43 --> mysqli_sql_exception: Duplicate entry 'gabrielvitorsilvabrito@gmail.com' for key 'usuarios_email' in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\codeigniter4\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(750): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\codeigniter4\app\Controllers\Usuarios.php(113): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Usuarios->create()
#10 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#11 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\codeigniter4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\xampp\htdocs\codeigniter4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#16 {main}
CRITICAL - 2025-06-05 17:16:43 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'gabrielvitorsilvabrito@gmail.com' for key 'usuarios_email'
[Method: POST, Route: usuarios/create]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (:usuarios_nome:, :usuarios_sobrenome:, :usuarios_email:, :usuarios_cpf:, :usuarios_data_nasc:, :usuarios_senha:, :usuarios_fone:, :usuarios_nivel:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Usuarios.php(113): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->create()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-05 17:16:43 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'gabrielvitorsilvabrito@gmail.com' for key 'usuarios_email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Isaque\', \'Pontes\', \'gabrielvitorsilvabrito@gmail.com\', \'71173068155\', \'2004-03-19\', \'202cb962ac59075b964b07152d234b70\', \'62986073404\', 0)')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Isaque\', \'Pontes\', \'gabrielvitorsilvabrito@gmail.com\', \'71173068155\', \'2004-03-19\', \'202cb962ac59075b964b07152d234b70\', \'62986073404\', 0)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (:usuarios_nome:, :usuarios_sobrenome:, :usuarios_email:, :usuarios_cpf:, :usuarios_data_nasc:, :usuarios_senha:, :usuarios_fone:, :usuarios_nivel:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Usuarios.php(113): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->create()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-05 17:16:43 --> [Caused by] mysqli_sql_exception: Duplicate entry 'gabrielvitorsilvabrito@gmail.com' for key 'usuarios_email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Isaque\', \'Pontes\', \'gabrielvitorsilvabrito@gmail.com\', \'71173068155\', \'2004-03-19\', \'202cb962ac59075b964b07152d234b70\', \'62986073404\', 0)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Isaque\', \'Pontes\', \'gabrielvitorsilvabrito@gmail.com\', \'71173068155\', \'2004-03-19\', \'202cb962ac59075b964b07152d234b70\', \'62986073404\', 0)')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Isaque\', \'Pontes\', \'gabrielvitorsilvabrito@gmail.com\', \'71173068155\', \'2004-03-19\', \'202cb962ac59075b964b07152d234b70\', \'62986073404\', 0)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (:usuarios_nome:, :usuarios_sobrenome:, :usuarios_email:, :usuarios_cpf:, :usuarios_data_nasc:, :usuarios_senha:, :usuarios_fone:, :usuarios_nivel:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Usuarios.php(113): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->create()
11 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
12 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
16 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 17:16:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:17:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:17:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 17:17:05 --> ErrorException: Undefined array key 0
[Method: GET, Route: usuarios/edit/assets/css/style.css]
in APPPATH\Controllers\Usuarios.php on line 157.
 1 APPPATH\Controllers\Usuarios.php(157): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 0', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Usuarios.php', 157)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->edit('assets', 'css', 'style.css')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
ERROR - 2025-06-05 17:17:09 --> mysqli_sql_exception: Duplicate entry 'gabrielvitorsilvabrito@gmail.com' for key 'usuarios_email' in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\codeigniter4\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(750): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\codeigniter4\app\Controllers\Usuarios.php(113): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Usuarios->create()
#10 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#11 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\codeigniter4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\xampp\htdocs\codeigniter4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#16 {main}
CRITICAL - 2025-06-05 17:17:09 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'gabrielvitorsilvabrito@gmail.com' for key 'usuarios_email'
[Method: POST, Route: usuarios/create]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (:usuarios_nome:, :usuarios_sobrenome:, :usuarios_email:, :usuarios_cpf:, :usuarios_data_nasc:, :usuarios_senha:, :usuarios_fone:, :usuarios_nivel:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Usuarios.php(113): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->create()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-05 17:17:09 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'gabrielvitorsilvabrito@gmail.com' for key 'usuarios_email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Isaque\', \'Pontes\', \'gabrielvitorsilvabrito@gmail.com\', \'71173068155\', \'2004-03-19\', \'d9b1d7db4cd6e70935368a1efb10e377\', \'62986073404\', 0)')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Isaque\', \'Pontes\', \'gabrielvitorsilvabrito@gmail.com\', \'71173068155\', \'2004-03-19\', \'d9b1d7db4cd6e70935368a1efb10e377\', \'62986073404\', 0)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (:usuarios_nome:, :usuarios_sobrenome:, :usuarios_email:, :usuarios_cpf:, :usuarios_data_nasc:, :usuarios_senha:, :usuarios_fone:, :usuarios_nivel:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Usuarios.php(113): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->create()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-05 17:17:09 --> [Caused by] mysqli_sql_exception: Duplicate entry 'gabrielvitorsilvabrito@gmail.com' for key 'usuarios_email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Isaque\', \'Pontes\', \'gabrielvitorsilvabrito@gmail.com\', \'71173068155\', \'2004-03-19\', \'d9b1d7db4cd6e70935368a1efb10e377\', \'62986073404\', 0)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Isaque\', \'Pontes\', \'gabrielvitorsilvabrito@gmail.com\', \'71173068155\', \'2004-03-19\', \'d9b1d7db4cd6e70935368a1efb10e377\', \'62986073404\', 0)')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Isaque\', \'Pontes\', \'gabrielvitorsilvabrito@gmail.com\', \'71173068155\', \'2004-03-19\', \'d9b1d7db4cd6e70935368a1efb10e377\', \'62986073404\', 0)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (:usuarios_nome:, :usuarios_sobrenome:, :usuarios_email:, :usuarios_cpf:, :usuarios_data_nasc:, :usuarios_senha:, :usuarios_fone:, :usuarios_nivel:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Usuarios.php(113): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->create()
11 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
12 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
16 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
ERROR - 2025-06-05 17:17:33 --> mysqli_sql_exception: Duplicate entry 'gabrielvitorsilvabrito@gmail.com' for key 'usuarios_email' in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\codeigniter4\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(750): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\codeigniter4\app\Controllers\Usuarios.php(113): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Usuarios->create()
#10 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#11 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\codeigniter4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\xampp\htdocs\codeigniter4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#16 {main}
CRITICAL - 2025-06-05 17:17:33 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'gabrielvitorsilvabrito@gmail.com' for key 'usuarios_email'
[Method: POST, Route: usuarios/create]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (:usuarios_nome:, :usuarios_sobrenome:, :usuarios_email:, :usuarios_cpf:, :usuarios_data_nasc:, :usuarios_senha:, :usuarios_fone:, :usuarios_nivel:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Usuarios.php(113): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->create()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-05 17:17:33 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'gabrielvitorsilvabrito@gmail.com' for key 'usuarios_email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Isaque\', \'Pontes\', \'gabrielvitorsilvabrito@gmail.com\', \'hdwejfw\', \'2004-03-19\', \'d9b1d7db4cd6e70935368a1efb10e377\', \'62986073404\', 0)')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Isaque\', \'Pontes\', \'gabrielvitorsilvabrito@gmail.com\', \'hdwejfw\', \'2004-03-19\', \'d9b1d7db4cd6e70935368a1efb10e377\', \'62986073404\', 0)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (:usuarios_nome:, :usuarios_sobrenome:, :usuarios_email:, :usuarios_cpf:, :usuarios_data_nasc:, :usuarios_senha:, :usuarios_fone:, :usuarios_nivel:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Usuarios.php(113): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->create()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-05 17:17:33 --> [Caused by] mysqli_sql_exception: Duplicate entry 'gabrielvitorsilvabrito@gmail.com' for key 'usuarios_email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Isaque\', \'Pontes\', \'gabrielvitorsilvabrito@gmail.com\', \'hdwejfw\', \'2004-03-19\', \'d9b1d7db4cd6e70935368a1efb10e377\', \'62986073404\', 0)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Isaque\', \'Pontes\', \'gabrielvitorsilvabrito@gmail.com\', \'hdwejfw\', \'2004-03-19\', \'d9b1d7db4cd6e70935368a1efb10e377\', \'62986073404\', 0)')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Isaque\', \'Pontes\', \'gabrielvitorsilvabrito@gmail.com\', \'hdwejfw\', \'2004-03-19\', \'d9b1d7db4cd6e70935368a1efb10e377\', \'62986073404\', 0)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (:usuarios_nome:, :usuarios_sobrenome:, :usuarios_email:, :usuarios_cpf:, :usuarios_data_nasc:, :usuarios_senha:, :usuarios_fone:, :usuarios_nivel:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Usuarios.php(113): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->create()
11 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
12 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
16 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
ERROR - 2025-06-05 17:17:53 --> mysqli_sql_exception: Duplicate entry 'gabrielvitorsilvabrito@gmail.com' for key 'usuarios_email' in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\codeigniter4\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(750): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\codeigniter4\app\Controllers\Usuarios.php(113): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Usuarios->create()
#10 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#11 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\codeigniter4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\xampp\htdocs\codeigniter4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#16 {main}
CRITICAL - 2025-06-05 17:17:53 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'gabrielvitorsilvabrito@gmail.com' for key 'usuarios_email'
[Method: POST, Route: usuarios/create]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (:usuarios_nome:, :usuarios_sobrenome:, :usuarios_email:, :usuarios_cpf:, :usuarios_data_nasc:, :usuarios_senha:, :usuarios_fone:, :usuarios_nivel:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Usuarios.php(113): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->create()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-05 17:17:53 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'gabrielvitorsilvabrito@gmail.com' for key 'usuarios_email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Isaque\', \'Pontes\', \'gabrielvitorsilvabrito@gmail.com\', \'33456\', \'2004-03-19\', \'d9b1d7db4cd6e70935368a1efb10e377\', \'62986073404\', 0)')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Isaque\', \'Pontes\', \'gabrielvitorsilvabrito@gmail.com\', \'33456\', \'2004-03-19\', \'d9b1d7db4cd6e70935368a1efb10e377\', \'62986073404\', 0)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (:usuarios_nome:, :usuarios_sobrenome:, :usuarios_email:, :usuarios_cpf:, :usuarios_data_nasc:, :usuarios_senha:, :usuarios_fone:, :usuarios_nivel:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Usuarios.php(113): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->create()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-05 17:17:53 --> [Caused by] mysqli_sql_exception: Duplicate entry 'gabrielvitorsilvabrito@gmail.com' for key 'usuarios_email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Isaque\', \'Pontes\', \'gabrielvitorsilvabrito@gmail.com\', \'33456\', \'2004-03-19\', \'d9b1d7db4cd6e70935368a1efb10e377\', \'62986073404\', 0)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Isaque\', \'Pontes\', \'gabrielvitorsilvabrito@gmail.com\', \'33456\', \'2004-03-19\', \'d9b1d7db4cd6e70935368a1efb10e377\', \'62986073404\', 0)')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Isaque\', \'Pontes\', \'gabrielvitorsilvabrito@gmail.com\', \'33456\', \'2004-03-19\', \'d9b1d7db4cd6e70935368a1efb10e377\', \'62986073404\', 0)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (:usuarios_nome:, :usuarios_sobrenome:, :usuarios_email:, :usuarios_cpf:, :usuarios_data_nasc:, :usuarios_senha:, :usuarios_fone:, :usuarios_nivel:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Usuarios.php(113): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->create()
11 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
12 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
16 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 17:17:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:18:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 17:18:03 --> ErrorException: Undefined array key 0
[Method: GET, Route: usuarios/edit/assets/css/style.css]
in APPPATH\Controllers\Usuarios.php on line 157.
 1 APPPATH\Controllers\Usuarios.php(157): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 0', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Usuarios.php', 157)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->edit('assets', 'css', 'style.css')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 17:26:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:26:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:26:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:26:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:26:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:26:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:26:28 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:28:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:36:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 17:36:50 --> ErrorException: Undefined array key 0
[Method: GET, Route: usuarios/edit/assets/css/style.css]
in APPPATH\Controllers\Usuarios.php on line 157.
 1 APPPATH\Controllers\Usuarios.php(157): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 0', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Usuarios.php', 157)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->edit('assets', 'css', 'style.css')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
ERROR - 2025-06-05 17:36:54 --> mysqli_sql_exception: Duplicate entry 'vilsonsoares@gmail.com' for key 'usuarios_email' in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\codeigniter4\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(750): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\codeigniter4\app\Controllers\Usuarios.php(113): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Usuarios->create()
#10 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#11 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\codeigniter4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\xampp\htdocs\codeigniter4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#16 {main}
CRITICAL - 2025-06-05 17:36:54 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'vilsonsoares@gmail.com' for key 'usuarios_email'
[Method: POST, Route: usuarios/create]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (:usuarios_nome:, :usuarios_sobrenome:, :usuarios_email:, :usuarios_cpf:, :usuarios_data_nasc:, :usuarios_senha:, :usuarios_fone:, :usuarios_nivel:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Usuarios.php(113): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->create()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-05 17:36:54 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'vilsonsoares@gmail.com' for key 'usuarios_email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Vilson\', \'Soares de Siqueira\', \'vilsonsoares@gmail.com\', \'999.999.999-99\', \'1981-12-03\', \'14e1b600b1fd579f47433b88e8d85291\', \'6398474-3380\', 0)')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Vilson\', \'Soares de Siqueira\', \'vilsonsoares@gmail.com\', \'999.999.999-99\', \'1981-12-03\', \'14e1b600b1fd579f47433b88e8d85291\', \'6398474-3380\', 0)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (:usuarios_nome:, :usuarios_sobrenome:, :usuarios_email:, :usuarios_cpf:, :usuarios_data_nasc:, :usuarios_senha:, :usuarios_fone:, :usuarios_nivel:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Usuarios.php(113): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->create()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-05 17:36:54 --> [Caused by] mysqli_sql_exception: Duplicate entry 'vilsonsoares@gmail.com' for key 'usuarios_email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Vilson\', \'Soares de Siqueira\', \'vilsonsoares@gmail.com\', \'999.999.999-99\', \'1981-12-03\', \'14e1b600b1fd579f47433b88e8d85291\', \'6398474-3380\', 0)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Vilson\', \'Soares de Siqueira\', \'vilsonsoares@gmail.com\', \'999.999.999-99\', \'1981-12-03\', \'14e1b600b1fd579f47433b88e8d85291\', \'6398474-3380\', 0)')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Vilson\', \'Soares de Siqueira\', \'vilsonsoares@gmail.com\', \'999.999.999-99\', \'1981-12-03\', \'14e1b600b1fd579f47433b88e8d85291\', \'6398474-3380\', 0)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (:usuarios_nome:, :usuarios_sobrenome:, :usuarios_email:, :usuarios_cpf:, :usuarios_data_nasc:, :usuarios_senha:, :usuarios_fone:, :usuarios_nivel:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Usuarios.php(113): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->create()
11 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
12 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
16 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 17:37:07 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 17:37:07 --> ErrorException: Undefined array key 0
[Method: GET, Route: usuarios/edit/assets/css/style.css]
in APPPATH\Controllers\Usuarios.php on line 157.
 1 APPPATH\Controllers\Usuarios.php(157): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 0', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Usuarios.php', 157)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->edit('assets', 'css', 'style.css')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 17:37:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:37:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:37:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-06-05 17:39:55 --> mysqli_sql_exception: Duplicate entry 'asdhasfjhdfjk@gmail.com' for key 'usuarios_email' in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\codeigniter4\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(750): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\codeigniter4\app\Controllers\Usuarios.php(74): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Usuarios->create()
#10 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#11 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\codeigniter4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\xampp\htdocs\codeigniter4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#16 {main}
CRITICAL - 2025-06-05 17:39:55 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'asdhasfjhdfjk@gmail.com' for key 'usuarios_email'
[Method: POST, Route: usuarios/create]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (:usuarios_nome:, :usuarios_sobrenome:, :usuarios_email:, :usuarios_cpf:, :usuarios_data_nasc:, :usuarios_senha:, :usuarios_fone:, :usuarios_nivel:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Usuarios.php(74): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->create()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-05 17:39:55 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'asdhasfjhdfjk@gmail.com' for key 'usuarios_email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Gabriel Vítor\', \'Brito\', \'asdhasfjhdfjk@gmail.com\', \'123456789\', \'2004-03-19\', \'e10adc3949ba59abbe56e057f20f883e\', \'62986073404\', 0)')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Gabriel Vítor\', \'Brito\', \'asdhasfjhdfjk@gmail.com\', \'123456789\', \'2004-03-19\', \'e10adc3949ba59abbe56e057f20f883e\', \'62986073404\', 0)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (:usuarios_nome:, :usuarios_sobrenome:, :usuarios_email:, :usuarios_cpf:, :usuarios_data_nasc:, :usuarios_senha:, :usuarios_fone:, :usuarios_nivel:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Usuarios.php(74): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->create()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-05 17:39:55 --> [Caused by] mysqli_sql_exception: Duplicate entry 'asdhasfjhdfjk@gmail.com' for key 'usuarios_email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Gabriel Vítor\', \'Brito\', \'asdhasfjhdfjk@gmail.com\', \'123456789\', \'2004-03-19\', \'e10adc3949ba59abbe56e057f20f883e\', \'62986073404\', 0)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Gabriel Vítor\', \'Brito\', \'asdhasfjhdfjk@gmail.com\', \'123456789\', \'2004-03-19\', \'e10adc3949ba59abbe56e057f20f883e\', \'62986073404\', 0)')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Gabriel Vítor\', \'Brito\', \'asdhasfjhdfjk@gmail.com\', \'123456789\', \'2004-03-19\', \'e10adc3949ba59abbe56e057f20f883e\', \'62986073404\', 0)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (:usuarios_nome:, :usuarios_sobrenome:, :usuarios_email:, :usuarios_cpf:, :usuarios_data_nasc:, :usuarios_senha:, :usuarios_fone:, :usuarios_nivel:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Usuarios.php(74): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->create()
11 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
12 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
16 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
ERROR - 2025-06-05 17:40:08 --> mysqli_sql_exception: Duplicate entry 'asdhasfjhdfjk@gmail.com' for key 'usuarios_email' in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\codeigniter4\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(750): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\codeigniter4\app\Controllers\Usuarios.php(74): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Usuarios->create()
#10 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#11 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\codeigniter4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\xampp\htdocs\codeigniter4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#16 {main}
CRITICAL - 2025-06-05 17:40:08 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'asdhasfjhdfjk@gmail.com' for key 'usuarios_email'
[Method: POST, Route: usuarios/create]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (:usuarios_nome:, :usuarios_sobrenome:, :usuarios_email:, :usuarios_cpf:, :usuarios_data_nasc:, :usuarios_senha:, :usuarios_fone:, :usuarios_nivel:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Usuarios.php(74): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->create()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-05 17:40:08 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'asdhasfjhdfjk@gmail.com' for key 'usuarios_email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Gabriel Vítor\', \'Brito\', \'asdhasfjhdfjk@gmail.com\', \'123456789\', \'2004-03-19\', \'e10adc3949ba59abbe56e057f20f883e\', \'62986073404\', 0)')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Gabriel Vítor\', \'Brito\', \'asdhasfjhdfjk@gmail.com\', \'123456789\', \'2004-03-19\', \'e10adc3949ba59abbe56e057f20f883e\', \'62986073404\', 0)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (:usuarios_nome:, :usuarios_sobrenome:, :usuarios_email:, :usuarios_cpf:, :usuarios_data_nasc:, :usuarios_senha:, :usuarios_fone:, :usuarios_nivel:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Usuarios.php(74): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->create()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-05 17:40:08 --> [Caused by] mysqli_sql_exception: Duplicate entry 'asdhasfjhdfjk@gmail.com' for key 'usuarios_email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Gabriel Vítor\', \'Brito\', \'asdhasfjhdfjk@gmail.com\', \'123456789\', \'2004-03-19\', \'e10adc3949ba59abbe56e057f20f883e\', \'62986073404\', 0)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Gabriel Vítor\', \'Brito\', \'asdhasfjhdfjk@gmail.com\', \'123456789\', \'2004-03-19\', \'e10adc3949ba59abbe56e057f20f883e\', \'62986073404\', 0)')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Gabriel Vítor\', \'Brito\', \'asdhasfjhdfjk@gmail.com\', \'123456789\', \'2004-03-19\', \'e10adc3949ba59abbe56e057f20f883e\', \'62986073404\', 0)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (:usuarios_nome:, :usuarios_sobrenome:, :usuarios_email:, :usuarios_cpf:, :usuarios_data_nasc:, :usuarios_senha:, :usuarios_fone:, :usuarios_nivel:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Usuarios.php(74): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->create()
11 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
12 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
16 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 17:40:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:40:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 17:40:13 --> ErrorException: Undefined variable $cidades
[Method: GET, Route: usuarios/new]
in APPPATH\Views\usuarios\form.php on line 75.
 1 APPPATH\Views\usuarios\form.php(75): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $cidades', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\usuarios\\form.php', 75)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\usuarios\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('usuarios/form', [], true)
 5 APPPATH\Controllers\Usuarios.php(39): view('usuarios/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->new()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 17:42:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:42:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:42:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:42:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 17:43:46 --> ErrorException: Undefined array key 0
[Method: POST, Route: usuarios/create]
in APPPATH\Controllers\Usuarios.php on line 89.
 1 APPPATH\Controllers\Usuarios.php(89): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 0', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Usuarios.php', 89)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->create()
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
ERROR - 2025-06-05 17:45:03 --> mysqli_sql_exception: Duplicate entry 'xgajxnAJSJLSAJ@gmail.com' for key 'usuarios_email' in C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\codeigniter4\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `us...', 0)
#1 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `us...')
#2 C:\xampp\htdocs\codeigniter4\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `us...')
#3 C:\xampp\htdocs\codeigniter4\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `us...', Array, false)
#4 C:\xampp\htdocs\codeigniter4\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(839): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\codeigniter4\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, false)
#7 C:\xampp\htdocs\codeigniter4\system\BaseModel.php(750): CodeIgniter\Model->insert(Array, false)
#8 C:\xampp\htdocs\codeigniter4\app\Controllers\Usuarios.php(78): CodeIgniter\BaseModel->save(Array)
#9 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(933): App\Controllers\Usuarios->create()
#10 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
#11 C:\xampp\htdocs\codeigniter4\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 C:\xampp\htdocs\codeigniter4\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#13 C:\xampp\htdocs\codeigniter4\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#14 C:\xampp\htdocs\codeigniter4\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#15 C:\xampp\htdocs\codeigniter4\system\rewrite.php(44): require_once('C:\\xampp\\htdocs...')
#16 {main}
CRITICAL - 2025-06-05 17:45:03 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'xgajxnAJSJLSAJ@gmail.com' for key 'usuarios_email'
[Method: POST, Route: usuarios/create]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (:usuarios_nome:, :usuarios_sobrenome:, :usuarios_email:, :usuarios_cpf:, :usuarios_data_nasc:, :usuarios_senha:, :usuarios_fone:, :usuarios_nivel:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 5 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->insert([...], false)
 6 APPPATH\Controllers\Usuarios.php(78): CodeIgniter\BaseModel->save([...])
 7 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->create()
 8 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 9 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
10 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
11 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
12 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
13 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-05 17:45:03 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry 'xgajxnAJSJLSAJ@gmail.com' for key 'usuarios_email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Teste\', \'Da Silva\', \'xgajxnAJSJLSAJ@gmail.com\', \'34567890988\', \'2004-03-19\', \'e10adc3949ba59abbe56e057f20f883e\', \'62986073404\', 0)')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Teste\', \'Da Silva\', \'xgajxnAJSJLSAJ@gmail.com\', \'34567890988\', \'2004-03-19\', \'e10adc3949ba59abbe56e057f20f883e\', \'62986073404\', 0)')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (:usuarios_nome:, :usuarios_sobrenome:, :usuarios_email:, :usuarios_cpf:, :usuarios_data_nasc:, :usuarios_senha:, :usuarios_fone:, :usuarios_nivel:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 7 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->insert([...], false)
 8 APPPATH\Controllers\Usuarios.php(78): CodeIgniter\BaseModel->save([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->create()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
15 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-05 17:45:03 --> [Caused by] mysqli_sql_exception: Duplicate entry 'xgajxnAJSJLSAJ@gmail.com' for key 'usuarios_email'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Teste\', \'Da Silva\', \'xgajxnAJSJLSAJ@gmail.com\', \'34567890988\', \'2004-03-19\', \'e10adc3949ba59abbe56e057f20f883e\', \'62986073404\', 0)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Teste\', \'Da Silva\', \'xgajxnAJSJLSAJ@gmail.com\', \'34567890988\', \'2004-03-19\', \'e10adc3949ba59abbe56e057f20f883e\', \'62986073404\', 0)')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (\'Teste\', \'Da Silva\', \'xgajxnAJSJLSAJ@gmail.com\', \'34567890988\', \'2004-03-19\', \'e10adc3949ba59abbe56e057f20f883e\', \'62986073404\', 0)')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `usuarios` (`usuarios_nome`, `usuarios_sobrenome`, `usuarios_email`, `usuarios_cpf`, `usuarios_data_nasc`, `usuarios_senha`, `usuarios_fone`, `usuarios_nivel`) VALUES (:usuarios_nome:, :usuarios_sobrenome:, :usuarios_email:, :usuarios_cpf:, :usuarios_data_nasc:, :usuarios_senha:, :usuarios_fone:, :usuarios_nivel:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(839): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], false)
 8 SYSTEMPATH\BaseModel.php(750): CodeIgniter\Model->insert([...], false)
 9 APPPATH\Controllers\Usuarios.php(78): CodeIgniter\BaseModel->save([...])
10 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->create()
11 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
12 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
14 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
15 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
16 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 17:45:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:45:29 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:45:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:45:36 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 17:45:37 --> ErrorException: Undefined array key 0
[Method: GET, Route: usuarios/edit/assets/css/style.css]
in APPPATH\Controllers\Usuarios.php on line 119.
 1 APPPATH\Controllers\Usuarios.php(119): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 0', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Usuarios.php', 119)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->edit('assets', 'css', 'style.css')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 17:45:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:45:48 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 17:45:48 --> ErrorException: Undefined array key 0
[Method: GET, Route: usuarios/edit/assets/css/style.css]
in APPPATH\Controllers\Usuarios.php on line 119.
 1 APPPATH\Controllers\Usuarios.php(119): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 0', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Usuarios.php', 119)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->edit('assets', 'css', 'style.css')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 17:45:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:45:57 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:47:08 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:47:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:47:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:47:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:47:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:47:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:47:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:47:17 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:47:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:49:20 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:49:34 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:50:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:50:26 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 17:50:26 --> ErrorException: Attempt to read property "enderecos_id" on null
[Method: GET, Route: enderecos/edit/assets/css/style.css]
in APPPATH\Views\enderecos\form.php on line 22.
 1 APPPATH\Views\enderecos\form.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "enderecos_id" on null', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\enderecos\\form.php', 22)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\enderecos\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('enderecos/form', [], true)
 5 APPPATH\Controllers\Enderecos.php(157): view('enderecos/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Enderecos->edit('assets', 'css', 'style.css')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Enderecos))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 17:50:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:50:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:50:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 17:50:39 --> ErrorException: Attempt to read property "enderecos_id" on null
[Method: GET, Route: enderecos/edit/assets/css/style.css]
in APPPATH\Views\enderecos\form.php on line 22.
 1 APPPATH\Views\enderecos\form.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "enderecos_id" on null', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\enderecos\\form.php', 22)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\enderecos\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('enderecos/form', [], true)
 5 APPPATH\Controllers\Enderecos.php(157): view('enderecos/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Enderecos->edit('assets', 'css', 'style.css')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Enderecos))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 17:50:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:50:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:50:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:50:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 17:50:56 --> ErrorException: Attempt to read property "enderecos_id" on null
[Method: GET, Route: enderecos/edit/assets/css/style.css]
in APPPATH\Views\enderecos\form.php on line 22.
 1 APPPATH\Views\enderecos\form.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "enderecos_id" on null', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\enderecos\\form.php', 22)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\enderecos\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('enderecos/form', [], true)
 5 APPPATH\Controllers\Enderecos.php(157): view('enderecos/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Enderecos->edit('assets', 'css', 'style.css')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Enderecos))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 17:55:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:55:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 17:55:39 --> ErrorException: Attempt to read property "enderecos_id" on null
[Method: GET, Route: enderecos/edit/assets/css/style.css]
in APPPATH\Views\enderecos\form.php on line 22.
 1 APPPATH\Views\enderecos\form.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "enderecos_id" on null', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\enderecos\\form.php', 22)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\enderecos\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('enderecos/form', [], true)
 5 APPPATH\Controllers\Enderecos.php(157): view('enderecos/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Enderecos->edit('assets', 'css', 'style.css')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Enderecos))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 17:55:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:55:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:55:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 17:55:49 --> ErrorException: Attempt to read property "enderecos_id" on null
[Method: GET, Route: enderecos/edit/assets/css/style.css]
in APPPATH\Views\enderecos\form.php on line 22.
 1 APPPATH\Views\enderecos\form.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "enderecos_id" on null', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\enderecos\\form.php', 22)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\enderecos\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('enderecos/form', [], true)
 5 APPPATH\Controllers\Enderecos.php(157): view('enderecos/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Enderecos->edit('assets', 'css', 'style.css')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Enderecos))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 17:55:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:55:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 17:55:54 --> ErrorException: Attempt to read property "enderecos_id" on null
[Method: GET, Route: enderecos/edit/assets/css/style.css]
in APPPATH\Views\enderecos\form.php on line 22.
 1 APPPATH\Views\enderecos\form.php(22): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "enderecos_id" on null', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\enderecos\\form.php', 22)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\enderecos\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('enderecos/form', [], true)
 5 APPPATH\Controllers\Enderecos.php(157): view('enderecos/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Enderecos->edit('assets', 'css', 'style.css')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Enderecos))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 17:55:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:55:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 17:55:59 --> ErrorException: Undefined array key 0
[Method: GET, Route: usuarios/edit/assets/css/style.css]
in APPPATH\Controllers\Usuarios.php on line 141.
 1 APPPATH\Controllers\Usuarios.php(141): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 0', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Usuarios.php', 141)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->edit('assets', 'css', 'style.css')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 17:57:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 17:57:11 --> ErrorException: Undefined array key 0
[Method: GET, Route: usuarios/edit/assets/css/style.css]
in APPPATH\Controllers\Usuarios.php on line 141.
 1 APPPATH\Controllers\Usuarios.php(141): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 0', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Usuarios.php', 141)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->edit('assets', 'css', 'style.css')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 17:57:14 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 17:57:16 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 17:57:16 --> ErrorException: Undefined array key 0
[Method: GET, Route: usuarios/edit/assets/css/style.css]
in APPPATH\Controllers\Usuarios.php on line 141.
 1 APPPATH\Controllers\Usuarios.php(141): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined array key 0', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Controllers\\Usuarios.php', 141)
 2 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->edit('assets', 'css', 'style.css')
 3 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 4 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 6 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 7 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 8 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 18:02:02 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 18:02:02 --> ErrorException: Undefined property: stdClass::$usuarios_logradouro
[Method: GET, Route: usuarios/edit/24]
in APPPATH\Views\usuarios\form.php on line 58.
 1 APPPATH\Views\usuarios\form.php(58): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$usuarios_logradouro', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\usuarios\\form.php', 58)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\usuarios\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('usuarios/form', [], true)
 5 APPPATH\Controllers\Usuarios.php(146): view('usuarios/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->edit('24')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 18:04:35 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 18:04:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 18:04:37 --> ErrorException: Undefined property: stdClass::$usuarios_logradouro
[Method: GET, Route: usuarios/edit/1]
in APPPATH\Views\usuarios\form.php on line 58.
 1 APPPATH\Views\usuarios\form.php(58): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$usuarios_logradouro', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\usuarios\\form.php', 58)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\usuarios\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('usuarios/form', [], true)
 5 APPPATH\Controllers\Usuarios.php(151): view('usuarios/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->edit('1')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 18:04:39 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 18:04:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 18:04:41 --> ErrorException: Undefined property: stdClass::$usuarios_logradouro
[Method: GET, Route: usuarios/edit/24]
in APPPATH\Views\usuarios\form.php on line 58.
 1 APPPATH\Views\usuarios\form.php(58): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$usuarios_logradouro', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\usuarios\\form.php', 58)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\usuarios\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('usuarios/form', [], true)
 5 APPPATH\Controllers\Usuarios.php(151): view('usuarios/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->edit('24')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 18:06:09 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 18:06:09 --> ErrorException: Undefined property: stdClass::$usuarios_logradouro
[Method: GET, Route: usuarios/edit/24]
in APPPATH\Views\usuarios\form.php on line 58.
 1 APPPATH\Views\usuarios\form.php(58): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$usuarios_logradouro', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\usuarios\\form.php', 58)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\usuarios\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('usuarios/form', [], true)
 5 APPPATH\Controllers\Usuarios.php(163): view('usuarios/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->edit('24')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 18:06:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 18:06:11 --> ErrorException: Undefined property: stdClass::$usuarios_logradouro
[Method: GET, Route: usuarios/edit/24]
in APPPATH\Views\usuarios\form.php on line 58.
 1 APPPATH\Views\usuarios\form.php(58): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$usuarios_logradouro', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\usuarios\\form.php', 58)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\usuarios\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('usuarios/form', [], true)
 5 APPPATH\Controllers\Usuarios.php(163): view('usuarios/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->edit('24')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 18:06:12 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 18:06:13 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 18:06:13 --> ErrorException: Undefined property: stdClass::$usuarios_logradouro
[Method: GET, Route: usuarios/edit/1]
in APPPATH\Views\usuarios\form.php on line 58.
 1 APPPATH\Views\usuarios\form.php(58): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$usuarios_logradouro', 'C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\usuarios\\form.php', 58)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\codeigniter4\\app\\Views\\usuarios\\form.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('usuarios/form', [], true)
 5 APPPATH\Controllers\Usuarios.php(163): view('usuarios/form', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->edit('1')
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
12 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 18:08:49 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 18:08:50 --> Error: Attempt to assign property "usuarios_logradouro" on null
[Method: GET, Route: usuarios/edit/assets/css/style.css]
in APPPATH\Controllers\Usuarios.php on line 153.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->edit('assets', 'css', 'style.css')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
CRITICAL - 2025-06-05 18:08:50 --> Error: Attempt to assign property "usuarios_logradouro" on null
[Method: GET, Route: usuarios/edit/assets/css/style.css]
in APPPATH\Controllers\Usuarios.php on line 153.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->edit('assets', 'css', 'style.css')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
DEBUG - 2025-06-05 18:08:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-06-05 18:08:56 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-06-05 18:08:56 --> Error: Attempt to assign property "usuarios_logradouro" on null
[Method: GET, Route: usuarios/edit/assets/css/style.css]
in APPPATH\Controllers\Usuarios.php on line 153.
 1 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Usuarios->edit('assets', 'css', 'style.css')
 2 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Usuarios))
 3 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
 5 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 6 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
 7 SYSTEMPATH\rewrite.php(44): require_once('C:\\xampp\\htdocs\\codeigniter4\\public\\index.php')
